### ISSUE TITLE
Issue with Notify Push

### ISSUE THREAD

**Issue by enoch85** [nextcloud/vm#1853]
```
It seems like notify_push is installed already, trying to enable it...
notify_push 0.1.4 enabled
✓ redis is configured
✓ push server is receiving redis messages
✓ push server can load mount info from database
✓ push server can connect to the Nextcloud server
🗴 push server is not a trusted proxy, please add '127.0.0.1' to the list of trusted proxies or configure any existing reverse proxy to forward the 'x-forwarded-for' send by the push server.
  If you're having issues getting the trusted proxy setup working, you can try bypassing any existing reverse proxy
  in your setup by setting the `NEXTCLOUD_URL` environment variable to point directly to the internal Nextcloud webserver url
  (You will still need the ip address of the push server added as trusted proxy)
```
cc @szaimen

**Comment by enoch85** [comment]
Fixed by reinstall, but shouldn't happen in the first place.

**Comment by Danteba** [comment]
Should the notify_push application install automatically with the upgrade to version 21?

**Comment by szaimen** [comment]
> Should the notify_push application install automatically with the upgrade to version 21?

I'd vote against it.
@enoch85 whats your opinion on that?

**Comment by enoch85** [comment]
> @enoch85 whats your opinion on that?

I'm thinking we will test this in NC 21, if we get no bug report we can at least choose it as a recommended app (marked YES) during the setup. AFAIK, it only works with TLS setup, so it can't be made default per say.

**Comment by szaimen** [comment]
This issue was fixed with #1854
